### DISCUSSION TITLE
Deprecated packages still in NPM

### DISCUSSION THREAD

**Discussion by Sugarcothe** [npm/feedback#459] [General]
Why is this site still hosting some deprecated packages, when they will not work

**Comment by ljharb** [comment]
Why wouldn't they work? "Deprecated" is both not "removed" nor necessarily "broken".

**Reply by Sugarcothe** [reply]
Thanks

**Comment by MylesBorins** [comment]
A package can be deprecated and still be relied on by large projects. While it is likely a good idea for projects to move off of them we can't remove those packages from the registry without causing significant damage.

**Reply by Sugarcothe** [reply]
Didn't get my head around that, thanks
